pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1265,12 +1292,24 @@ void GSQLBackend::getAllDomains(vector<DomainInfo> *domains, bool include_disabl
       } catch (...) {
         continue;
       }
+
+      if (pdns_iequals(row[3], "MASTER"))
+        di.kind = DomainInfo::Master;
+      else if (pdns_iequals(row[3], "SLAVE"))
+        di.kind = DomainInfo::Slave;
+      else
+        di.kind = DomainInfo::Native;
```

Comment:
Please add some logging if the type is not equal to 'NATIVE'

Reply:
now it will always log this even if the db already says "NATIVE"?